style.css

```diff
@@ -382,21 +427,355 @@ a.btn2 {
   padding-left: 10px;
 }
 
-.hi {
-  background: #1c1a19;
-  padding-top: 20px;
-  margin-top: -22px;
-  padding-bottom: 30px;
-  text-align: center;
-  padding-right: 50px;
+.foot {
+  justify-content: center;
+  padding: 12px 24px;
 }
 
-.di {
-  text-align: center;
-  font-family: "InaiMathi", sans-serif;
+.aa {
+  flex-wrap: wrap;
+  display: flex;
+  align-items: center;
+}
+
+.aa div {
+  justify-items: center;
+  flex-grow: 1;
+  flex-basis: 325px;
+}
+
+.ab {
+  justify-content: center;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: 700;
-  font-size: 32px;
+  font-size: 24px;
   line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  display: inline-block;
+}
+
+input {
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  border-bottom: 1px rgb(168, 166, 166) solid;
+  height: 30px;
+}
+
+.iam input {
+  width: 250px;
+}
```

Comment:
- [ ] [REQUIRED] Please implement design all inputs like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1278) because I see some flaws in your inputs. And please enter the `"First Name"` and `"Last Name"` inputs, because, in the template you have chosen, these are created as separate inputs!

![image](https://user-images.githubusercontent.com/63915024/186003740-7bc22aeb-7b6d-4f65-81e6-bbd3e60c02a0.png)

Reply:
Microverse told us specifically not to put a separate part for another name. We were asked to make only one input space for names

Reply:
![image](https://user-images.githubusercontent.com/109461921/186017698-86e18fa7-4a6d-4f21-8e26-850c865f22db.png)
we are nit meant to make the name field more than one